Farmacy/Models/Conversation.h

```diff
@@ -0,0 +1,17 @@
+//
+//  Conversation.h
+//  Farmacy
+//
+//  Created by Trang Dang on 8/1/22.
+//
+
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Conversation : PFObject <PFSubclassing>
+@property (nonatomic, strong) PFUser *userOne;
```

Comment:
Are userOne and userTwo for sender and receiver? If so, you can name them them sender and receiver. Thanks.

Reply:
Hi May! Conversation is between two users, both of them can be sender or receiver. That's why I cannot define who is sender or receiver in Conversation model. But by fetching all messages (Message model) that has Conversation ID along with message sender, I can know who sent this message (for example userOne), the another user of conversation (userTwo) will be receiver.